grpc/proto/v1/properties.proto

```diff
@@ -26,11 +26,49 @@ message Value {
     string blob_value = 10;
     PhoneNumber phone_value = 11;
     google.protobuf.NullValue null_value = 12;
+    string text_value = 13;
```

Comment:
why?

Reply:
Because I think the nomenclature of `string_value` was poorly chosen. The names here correspond to the allowed data types in Weaviate and `string` is deprecated in favour of `text`